test/library/packages/TOML/test/quoteTable.chpl

```diff
@@ -17,7 +24,13 @@ proc main() {
   const score1 = TomlData['LocalAtomics."0.1.0"']!['score']!.s;
   const score2 = TomlData['DataStructures."0.1.0"']!['score']!.s;
   const score3 = TomlData['GPUIterator."0.0.1"']!['score']!.s;
-  writeln(TomlData);
+  const score4 = TomlData['NumpyLike."0.1.0"']!['score']!.s;
+  const score5 = TomlData['csm."0.1.0"']!['score']!.s;
+  writeln(score1);
```

Comment:
Can you add an execopt and good file for `TomlTest.chpl` for this feature?

This version of the test is good, but we ought to have a version that prints the full TOML file as well (that test caught an issue not previously caught by this version of the test).

Note that you will have to remove the `key="value"` whitespace variants for the `TomlTest.chpl` version (so it's good that this test exercises those).

Reply:
@ben-albrecht I had added the execopts & good file. But, they don't match after the Toml is being parsed & I get the following output : 
```toml
[GnuPlot]

[GnuPlot."0]

[GnuPlot."0.0]

[GnuPlot."0.0.1"]
score = "4"

[LocalAtomics]

[LocalAtomics."0]

[LocalAtomics."0.1]

[LocalAtomics."0.1.0"]
score = "5"

[NumpyLike]

[NumpyLike."0]

[NumpyLike."0.0]

[NumpyLike."0.0.1"]
score = "2"

[csm]

[csm."0]

[csm."0.1]

[csm."0.1.0"]
score = "3"
```
However, we got the correct score of each package & test `quoteTable.chpl` passes.

Reply:
I think that's an error in the TOML module that needs to be addressed. I believe the output should be:

```toml
[GnuPlot]

[GnuPlot."0.0.1"]
score = "4"

...

```

Reply:
> @ben-albrecht made changes requested.

@ankingcodes - thanks. Did you see this comment?

Reply:
Oops, I missed that.  I had actually removed the good & execopts in 061c673.  However, is the Toml module error related to this issue and should be fixed in this PR ?

Reply:
>  is the Toml module error related to this issue and should be fixed in this PR ?

I lean towards yes. Storing the parsed TOML in the expected hierarchy is part of basic functionality IMO. If it turns out to be a large effort, we can add a future in this PR and follow up on it, but I'd prefer to tackle it here if possible.